ios/BT/ExposureManager.swift

```diff
@@ -525,4 +523,61 @@ private extension ExposureManager {
     localUncompressedURLs = []
     downloadedPackages = []
   }
+
+  func processExposureDetectionSummary(_ summary: ENExposureDetectionSummary?, completion: @escaping (([ExposureDetectionSummary]) -> Void)) {
+    if let summary = summary {
+
+      // Serialize exposure detection summary
+      let exposureDetectionSummary = ExposureDetectionSummary(summary)
+
+      // Prune exposure detection summaries >14 days from realm
+      BTSecureStorage.shared.pruneExposureDetectionSummaries()
+
+      // Fetch exposure summaries <14 days
+      let unusedExposureSummaries = BTSecureStorage.shared.allExposureDetectionSummaries()
+
+      // Store new exposure summary in realm
+      BTSecureStorage.shared.storeExposureDetectionSummary(exposureDetectionSummary)
+
+      // Call scoring server
+      apiClient.request(ExposureDetectionSummaryListRequest.post(exposureDetectionSummary,
+                                                                 unusedExposureSummaries),
+                        requestType: .scoring) { result in
+                          switch result {
+                          case .success(let scoringServerResponse):
+                            let summaries = scoringServerResponse.notifications.map { $0.exposureSummaries }.flatMap { $0 }
+                            completion(summaries)
+                          case .failure:
+                            completion([])
+                          }
+      }
+    }
+  }
+
+  func processScoringServerResponse(_ summaries: [ExposureDetectionSummary],
```

Comment:
Hmm, I'm a little confused here, but I also don't know Swift.

The first argument is the response from the server? Which should be a Notification, containing an array of summaries, but also other things.